### DISCUSSION TITLE
Figma UI Kit does not open

### DISCUSSION THREAD

**Discussion by AndreaOliveira** [primefaces/community#1941] [Figma UI Kit]
I bought a single license of the Figma UI Kit and it's not opening on my FIGMA. Does anyone know how to solve this?

**Comment by onursenture** [comment]
To use PrimeOne, you need to download Figma files from [PrimeStore](https://www.primefaces.org/store). First, log in to PrimeStore. Then, navigate to the UI Kit tab, download the zip file containing the Figma documents, and import its contents after unzipping into Figma. You can find a detailed guide here for import: https://help.figma.com/hc/en-us/articles/360041003114-Import-files-to-the-file-browser
